modules/tinymce/src/core/main/ts/keyboard/DeleteBackspaceKeys.ts

```diff
@@ -44,11 +44,11 @@ const executeKeydownOverride = (editor: Editor, caret: Cell<Text | null>, evt: K
     .filter((_) => editor.selection.isEditable())
     .each((applyAction) => {
       evt.preventDefault();
-      const beforeInput = fireFakeBeforeInputEvent(editor, inputType);
+      const beforeInput = fireBeforeInputEvent(editor, inputType);
```

Comment:
While you are at it maybe use namespace imports for this module everywhere it's being used.